baseapp/baseapp.go

```diff
@@ -22,7 +22,7 @@ import (
 // and to avoid affecting the Merkle root.
 var dbHeaderKey = []byte("header")
 
-// The ABCI application
+// BaseApp is an ABCI application.
```

Comment:
Unecessary. Our linting spec says this is fine. Please adhere: https://github.com/tendermint/lint